### PR TITLE
Add new keyboard Cornelius

### PR DESCRIPTION
**Pull request by foostan**
## Description

Add new keyboard `Cornelius`.
This keyboard is planned for the GB and is not yet available for sale.

geekhack -> [[IC] Cornelius - Gasket mounted 40% column staggered keyboard](https://geekhack.org/index.php?topic=109741.0)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/foostan/cornelius/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# cornelius
+
+![cornelius](https://i.imgur.com/kGoUGZg.jpg)
```
Comment (drashna) [inline keyboards/foostan/cornelius/readme.md:3]:
```suggestion
![cornelius](https://i.imgur.com/kGoUGZgl.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by foostan** [comment]
@drashna Would you mind confirming it again please?

File: keyboards/foostan/cornelius/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* Hardware Supported: *Cornelius PCBs*
+* Hardware Availability: *http://github.com/foostan*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foostan/cornelius:default
+
+Flashing example for this keyboard:
+
+    make foostan/cornelius:default:flash
```
Comment (drashna) [inline keyboards/foostan/cornelius/readme.md:17]: Could you add instructions on how to enter the bootloader somewhere here?

Comment (foostan) [reply keyboards/foostan/cornelius/readme.md:17]: Do you have an example anywhere?

Comment (drashna) [reply keyboards/foostan/cornelius/readme.md:17]: Something like: 
```
The board may be reset into bootloader mode via either the reset button (on the bottom of the PCB) or a configured RESET shortcut (which may vary depending on when the PCB was flashed).
```

File: keyboards/foostan/cornelius/config.h [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x3265
+#define PRODUCT_ID   0x0005
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Yushakobo
+#define PRODUCT      Cornelius
+#define DESCRIPTION  Gasket mounted 40% column staggered keyboard.
+
```
Comment (zvecr) [inline keyboards/foostan/cornelius/config.h:29]:
```suggestion

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/foostan/cornelius/readme.md (hunk 2 above)
Comment (foostan) [reply keyboards/foostan/cornelius/readme.md:17]: OK, I see.

Comment (foostan) [reply keyboards/foostan/cornelius/readme.md]: It's DONE 4428697

**Comment by foostan** [comment]
@drashna @zvecr The fix is DONE so would you check it please?

**Review by drashna — APPROVED** [review]

**Comment by foostan** [comment]
@zvecr No problem?

File: keyboards/foostan/cornelius/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,\
+     _______, _______, _______, _______,   MO(3), _______, _______,   MO(2), _______, _______, _______, _______ \
+  ),
+
+[3] = LAYOUT( \
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, _______, _______, _______,   MO(3), _______, _______,   MO(3), _______, _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/foostan/cornelius/keymaps/default/keymap.c:44]:
```suggestion
[0] = LAYOUT(
      KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,     KC_I,    KC_O,   KC_P,  KC_BSPC,
      KC_ESC,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,     KC_K,    KC_L, KC_SCLN, KC_QUOT,
     KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,  KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,
     XXXXXXX, KC_LCTL, KC_LGUI, KC_LALT,   MO(1),  KC_SPC,  KC_ENT,   MO(2),  KC_RALT, KC_RGUI, KC_RCTL, XXXXXXX
  ),

[1] = LAYOUT(
     _______,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_BSPC,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, _______, _______, _______,   MO(1), _______, _______,   MO(3), _______, _______, _______, _______
  ),

[2] = LAYOUT(
     _______, KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,
     _______, _______, _______, _______,   MO(3), _______, _______,   MO(2), _______, _______, _______, _______
  ),

[3] = LAYOUT(
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, _______, _______, _______,   MO(3), _______, _______,   MO(3), _______, _______, _______, _______
```

File: keyboards/foostan/cornelius/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,\
+     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
+  ),
+
+  [3] = LAYOUT( \
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/foostan/cornelius/keymaps/via/keymap.c:44]:
```suggestion
  [0] = LAYOUT(
      KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,     KC_I,    KC_O,   KC_P,  KC_BSPC,
      KC_ESC,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,     KC_K,    KC_L, KC_SCLN, KC_QUOT,
     KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,  KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,
     XXXXXXX, KC_LCTL, KC_LGUI, KC_LALT, FN_MO13,  KC_SPC,  KC_ENT, FN_MO23,  KC_RALT, KC_RGUI, KC_RCTL, XXXXXXX
  ),

  [1] = LAYOUT(
     _______,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_BSPC,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,   KC_UP,KC_RIGHT, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
  ),

  [2] = LAYOUT(
     _______, KC_EXLM,   KC_AT, KC_HASH,  KC_DLR, KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_MINS,  KC_EQL, KC_LCBR, KC_RCBR, KC_PIPE,  KC_GRV,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_UNDS, KC_PLUS, KC_LBRC, KC_RBRC, KC_BSLS, KC_TILD,
     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
  ),

  [3] = LAYOUT(
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
```

File: keyboards/foostan/cornelius/readme.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+* Hardware Supported: *Cornelius PCBs*
+* Hardware Availability: *http://github.com/foostan*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foostan/cornelius:via
+
+Flashing example for this keyboard:
+
+    make foostan/cornelius:via:flash
```
Comment (fauxpark) [inline keyboards/foostan/cornelius/readme.md:17]:
```suggestion
Gasket mounted 40% column staggered keyboard manufactured by Yushakobo

* Keyboard Maintainer: [foostan](https://github.com/foostan/)
* Hardware Supported: Cornelius PCBs
* Hardware Availability: http://github.com/foostan

Make example for this keyboard (after setting up your build environment):

    make foostan/cornelius:default

Flashing example for this keyboard:

    make foostan/cornelius:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/foostan/cornelius/readme.md (hunk 6 above)
Comment (foostan) [reply keyboards/foostan/cornelius/readme.md:17]: @fauxpark Can you give us a reason? I can't understand your suggestion.

Comment (foostan) [reply keyboards/foostan/cornelius/readme.md:17]: ah, I see your suggestion.
https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md

It is confusing to include `*` in the template. I thought it was correct to add `*`.

Comment (fauxpark) [reply keyboards/foostan/cornelius/readme.md]: The italics are to show it is placeholder text. It's recommended to preview the markdown first so you can see if you've made any formatting errors.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**